plugins/modules/svc.py

```diff
@@ -30,23 +30,24 @@
     required: true
   state:
     description:
-      - V(started)/V(stopped) are idempotent actions that will not run commands unless necessary. V(restarted) will always bounce the svc (svc
-        -t) and V(killed) will always bounce the svc (svc -k). V(reloaded) will send a sigusr1 (svc -1). V(once) will run a normally downed svc
-        once (svc -o), not really an idempotent operation.
+      - V(started)/V(stopped) are idempotent actions that will not run commands unless necessary.
+      - V(restarted) will always bounce the svc (svc -t) and V(killed) will always bounce the svc (svc -k).
+      - V(reloaded) will send a sigusr1 (svc -1).
+      - V(once) will run a normally downed svc once (svc -o), not really an idempotent operation.
     type: str
     choices: [killed, once, reloaded, restarted, started, stopped]
   downed:
     description:
-      - Should a 'down' file exist or not, if it exists it disables auto startup. Defaults to no. Downed does not imply stopped.
+      - Should a C(down) file exist or not, if it exists it disables auto startup. Defaults to C(no). Downed does not imply stopped.
```

Comment:
I think this is talking about a value for this option:
```suggestion
      - Should a C(down) file exist or not, if it exists it disables auto startup. Defaults to V(false). Downed does not imply stopped.
```